nautobot/extras/views.py

```diff
@@ -1165,6 +1165,17 @@ def get_required_permission(self):
     def get_extra_context(self, request, instance):
         job_result = instance.get_latest_sync()
 
+        if job_result is None:
+            job_result = {
+                "completed": "—",
+                "date_created": "—",
+                "date_done": "—",
+                "duration": "—",
+                "name": "—",
+                "status": "N/A",
+                "user": "—",
+            }
```

Comment:
Can we add a view unit test case covering the originally described issue with this view?

Reply:
Done! Let me know what you think.